frontend/app/components/encryptable/form.js

```diff
@@ -98,6 +140,53 @@ export default class Form extends BaseFormComponent {
     this.changeset.folder = folder;
   }
 
+  @action
+  removeField(value) {
```

Comment:
huh do you really need to do all this by hand?
seems overly complicated.

Reply:
We have todo all this, but I generalized it to 3 lines instead of this if else construct :disappointed_relieved:

Reply:
Good job!